supabase/functions/_backend/public/channel.ts

```diff
@@ -76,8 +76,8 @@ export async function get(c: Context, body: GetDevice, apikey: Database['public'
         created_by,
         updated_at,
         public,
-        disableAutoUpdateUnderNative,
-        disableAutoUpdate,
```

Comment:
Still issue here you cannot change what the public API returns

Reply:
fixed in the latest commit